kubectl-volsync/cmd/migration_create.go

```diff
@@ -36,18 +54,29 @@ var migrationCreateCmd = &cobra.Command{
 	and it sets up an associated ReplicationDestination that will be configured
 	to accept incoming transfers via rsync over ssh.
 	`)),
-	RunE: doMigrationCreate,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		r := &migrationCreate{
+			Command: *cmd,
+		}
+		return r.Run()
+	},
 	Args: validateMigrationCreate,
 }
 
 func init() {
 	migrationCmd.AddCommand(migrationCreateCmd)
 
-	migrationCreateCmd.Flags().String("accessmodes", "", "AccessModes of the PVC to create")
-	migrationCreateCmd.Flags().String("capacity", "", "capacity of the PVC to create")
+	migrationCreateCmd.Flags().String("copymethod", "Direct",
+		"Copy method used to copy the data from source to destination PVC viz: Direct, Snapshot")
+	migrationCreateCmd.Flags().String("accessmodes", "ReadWriteOnce",
+		"accessMode of the PVC to create. viz: ReadWriteOnce, ReadOnlyMany, ReadWriteMany, ReadWriteOncePod")
+	migrationCreateCmd.Flags().String("capacity", "", "size of the PVC to create in Gi ex: 10Gi")
```

Comment:
```suggestion
	migrationCreateCmd.Flags().String("capacity", "", "size of the PVC to create (ex: 10Gi)")
```
Other units are allowed (100Mi, 2Ti, etc)

Reply:
done